### ISSUE TITLE
Fix Android build

### ISSUE THREAD

**Issue by Sheph** [Sheph/TriggerTime#2]
It would be nice to fix android build and get the game working on Android again, todo:
* write android makefiles for libRocket, freetype, etc. i.e. all the stuff that was added after Android build got broken
* make sure touchscreen controller code still works... (it should)
* make LuaJIT compile for Android (particularly important for performance reasons)
* anything else ???

**Comment by Poussinou** [comment]
If everything is open-source, and if the build isn't too complicated, we could add it to [F-Droid](https://f-droid.org/)! :)

**Comment by Sheph** [comment]
Yes, everything is open-source, the build shouldn't be too complicated, i.e. it's a matter of writing makefiles and m.b. fixing some platform-specific stuff. I would do it myself, but I don't have much free time and even when I have I'm working on a new game of mine :)
